.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
This was useful while trying to get the app to build on my local machine. `jenv` uses it to manage which version of java it should use - our app uses v1.8 while android studio with blaze defaults to 11.0.11

Reply:
Maybe a question for @fortuna , shall we upgrade Java to a newer version considering we are approaching the end-of-support time of Java 8: https://endoflife.date/java.

Reply:
Yes, let's upgrade to the newest Java if we can. But in another PR